doctr/models/recognition/sar/pytorch.py

```diff
@@ -207,8 +208,14 @@ def __init__(
         self.feat_extractor.train()
 
         self.encoder = SAREncoder(out_shape[1], rnn_units, dropout_prob)
-        self.decoder = SARDecoder(rnn_units, self.max_length, len(self.vocab),
-                                  embedding_units, attention_units, dropout_prob=dropout_prob)
+        self.decoder = SARDecoder(
+            rnn_units,
+            self.max_length,
+            len(self.vocab),
+            embedding_units,
+            attention_units,
+            dropout_prob=dropout_prob
```

Comment:
missing a comma here (reason behind this being that if you add a new arg, if the comma is already there the diff will be 1 line, vs if you dont --> 2 lines :) )